aws/data_source_aws_vpc_peering_connections_test.go

```diff
@@ -0,0 +1,83 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceAwsVpcPeeringConnections_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
```

Comment:
Prefer `ParallelTest` 👍 

```suggestion
	resource.ParallelTest(t, resource.TestCase{
```